private_gpt/ui/ui.py

```diff
@@ -367,6 +371,15 @@ def _build_ui_blocks(self) -> gr.Blocks:
                         size="sm",
                         visible=settings().ui.delete_all_files_button_enabled,
                     )
+                    adjust_model_temperature = gr.components.Number(
+                        value=settings().llm.temperature,
+                        label="Model Temperature",
+                        interactive=True,
+                        visible=settings().ui.adjust_temperature_enabled,
+                        minimum=0.05,
+                        maximum=0.95,
+                    )
+
```

Comment:
I would suggest adding a button to be clearer. Right now, we have to press Enter to accept the change. It doesn't seem to me the most accessible function.